apps/storybook-react/stories/Shadows.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
```

Comment:
We will likely want to improve these stories and documentation in the future, but for now, we can use it as a testing sandbox. It's by no means the final product.